components/DesaDigital/Detail/index.vue

```diff
@@ -0,0 +1,240 @@
+            <td>{{ item.properties.literacy.training.data || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+      <jds-simple-table v-show="item.level === 3 || item.level === 4" class="desa-digital-detail__content-table">
+        <thead>
+          <tr>
+            <th>Tentang BUMDesa</th>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Sosial Media yang Dikelola</strong>
+            </td>
+            <td>{{ socialMedias }}</td>
+          </tr>
+          <tr>
+            <td><strong>BumDesa</strong></td>
+            <td>{{ item.properties.bumdes.bumdes.data || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Komoditas yang dikelola BUMDesa</strong></td>
+            <td>{{ item.properties.bumdes.commodity.data || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+      <jds-simple-table v-show="item.level === 4" class="desa-digital-detail__content-table">
+        <thead>
+          <tr>
+            <th>Potensi Desa</th>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Kategori Potensi Desa</strong>
+            </td>
+            <td>{{ potentials }}</td>
+          </tr>
+          <tr>
+            <td><strong>Keterangan</strong></td>
+            <td>{{ item.properties.potential.growth_potential || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+    </div>
+  </div>
+</template>
+
+<script>
+import IconArrowLeft from '~/assets/icons/arrow-left.svg?inline'
+export default {
+  name: 'ComponentDesaDigitalDetail',
+  components: { IconArrowLeft },
+  props: {
+    item: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      showEditVillage: false,
+      mItem: {}
+    }
+  },
+  computed: {
+    vehicles () {
+      return this.joinDatas(this.item.properties.facility.vehicle_access.data)
+    },
+    communities () {
+      return this.joinDatas(this.item.properties.literacy.community.data)
+    },
+    socialMedias () {
+      return this.joinDatas(this.item.properties.bumdes.social_media.data)
+    },
+    potentials () {
+      return this.joinDatas(this.item.properties.potential.data, this.item.properties.potential.other_potential)
+    }
+  },
+  watch: {
+    item: {
+      handler (val) {
+        this.mItem = {
+          ...val
+        }
+      },
+      immediate: true,
+      deep: true
+    }
+  },
+  methods: {
+    joinDatas (datas, otherDatas) {
+      const newOtherDatas = otherDatas === undefined || otherDatas === null ? '' : `, ${otherDatas}`
+      if (Array.isArray(datas)) {
+        return datas.join(', ') + newOtherDatas
+      } else {
+        return '-'
+      }
+    },
```

Comment:
minor:

According to google 🤣 a word `Data` is the plural form of something Latin, so the word is plural by default. So there is no word `Datas` to represent more than one data. Maybe we can rename the function to `joinData`

also, what is the data type of `datas` and `otherDatas`?

Reply:
🤣 My bad, I think data is singularly turned out to be prular. I will change it. Thanks ⭐ 

### Ask

the data type of `datas` is an array and the data type of `otherDatas` is the string

Reply:
if otherData is a string wouldn't it easier to just use [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)?

```
const otherData = undefined

console.log(otherData ?? ' ' )
```

Reply:
I used to be like that too lol 🤣

Reply:
this change can see in [here](https://github.com/jabardigitalservice/dedi-cms/pull/92/commits/ea6908fc27b827567dfbddbb36bb38367f9acb5b)